docs/user/high-availability.md

```diff
@@ -0,0 +1,130 @@
+---
+navTitle: High Availability mode
+---
+
+# High Availability mode
+
+High Availability mode allows you to run multiple copies of your Node-RED instance,
+with incoming work distributed between them.
+
+This is a [Preview Feature](https://flowforge.com/handbook/product/versioning/#preview-features)
+that is available with:
+
+ - FlowForge 1.8+ running with an EE license and the kubernetes driver
+ - FlowForge Cloud
+
+Within FlowForge Cloud it is currently free to use for all teams, but will
+become a chargable feature in a future release.
+
+## Restrictions
+
+As a Preview Feature, there are a number of restrictions that apply to HA mode.
```

Comment:
```suggestion

```

Reply:
I would suggest removing this sentence because, in my view, it implies that all limitations will be addressed when the feature is no longer in preview, which is not true for at least some of these restrictions.